### ISSUE TITLE
App Design

### ISSUE THREAD

**Issue by MackleMullzy** [CHeffernan087/AlanMulreadyFitness#3]
Layout, Colour scheme, navigation

![image](https://user-images.githubusercontent.com/63520809/79134215-3e81ef00-7da5-11ea-8f27-352f59f177a2.png)

![image](https://user-images.githubusercontent.com/63520809/79134274-56597300-7da5-11ea-923a-cbce848f748e.png)

![image](https://user-images.githubusercontent.com/63520809/79134286-5b1e2700-7da5-11ea-9f2b-d9dfbeba3093.png)

![image](https://user-images.githubusercontent.com/63520809/79134304-64a78f00-7da5-11ea-92a8-49f136d74991.png)

Examples of leading apps already on the market with features we can implement

**Comment by CHeffernan087** [comment]
These are screenshots from other apps which we are going to base our color scheme on ?

**Comment by MackleMullzy** [comment]
> These are screenshots from other apps which we are going to base our color scheme on ?

No, just the function of them, so I like the way MFP uses a profile picture as your avatar, but the app itself is overcrowded. Jefit strips it back but again the newsfeed style is often ignored. Then centr fit is clean looking (owned by Chris Hemsworth, so a lot of money went into it I'd imagine) but again has that newsfeed style that can leave you scrolling for ages to get what you want

**Comment by CHeffernan087** [comment]
Got you 👍 💯

**Comment by MackleMullzy** [comment]
Main functions at launch

Home: Shows current weight, week of program. Bottom section shows section similar to MFP for Home, Workout, My Profile and Settings

Workout: Shows Workouts (Chest and Triceps, Legs etc.) Click workout to view and start at the top or bottom. Will have video example for exercise

My Profile: Name, avatar, weight, age and gallery for check in pics

Settings: Direct Debit, measurement (kg/lb, cm/ft inch) contact trainer section and contact us section (contact us for app itself)

Will draw up example, put here and Luke and Matthew will go over and improve

**Comment by MackleMullzy** [comment]
Uploading these test images, it will go into greater detail as design comes on. I'm limited with design (Proficient in MS Paint), so these are very much baseline
![App Home](https://user-images.githubusercontent.com/63520809/79692744-bd23d400-825e-11ea-9bf3-c8c27cc57e4f.png)
![My Profile](https://user-images.githubusercontent.com/63520809/79692745-bdbc6a80-825e-11ea-9881-f1a6b2c9b8d2.png)
![Settings](https://user-images.githubusercontent.com/63520809/79692746-be550100-825e-11ea-8b62-7de1d1c81716.png)
![Workout](https://user-images.githubusercontent.com/63520809/79692747-be550100-825e-11ea-898e-68f7324a5a99.png)

Again lads, open this up to fellow gym goers. Keep it under wraps but ask "What would you want to see in a fitness"

Right now the integration of MFP is a big one, but a bit down the line, as is Bluetooth device sync ( if possible at all)
